docs/06-concepts/06-database/05-crud.md

```diff
@@ -128,15 +128,15 @@ Update a single row, the following code will update the company name, but will n
 var company = await Company.db.findById(session, companyId); 
 company.name = 'New name';
 company.address = 'Baker street'
```

Comment:
Would you mind adding the missing semicolon here as well? 🙏

Reply:
of course! i'm still a n00b at dart/flutter/serverpod, so i appreciate the help, and the opportunity to help.